it_IT/readme.md

```diff
@@ -0,0 +1,951 @@
+
+Generalmente, i test eseguiti in serie creano cartelle temporanee nella cartella di test corrente per poi eliminarla al termine. Questo approccio non funziona quando i testi sono eseguiti concorrentemente poichè i test potrebbero entrare in conflitto l'uno con l'altro. L'approccio migliore per questo tipo di problema è quello di utilizzare una nuova cartella temporanea per ciascun file di test. I moduli [`tempfile`](https://github.com/sindresorhus/tempfile) e [`temp-write`](https://github.com/sindresorhus/temp-write) possono aiutare in questo.
+
+### Debugging
+
+AVA esegue i test in concorrenza in automatico, il che lo rende sub-ottimale quando devi fare il debug di qualcosa. In questo caso esegui i test in serie con l'opzione `--serial`:
+
+```
+$ ava --serial
+```
+
+### Code coverage
+
+Non è possibile utilizzare [`istanbul`](https://github.com/gotwarlost/istanbul) per il code coverage dato l'[uso di processi isolati](##isolamento-dei-processi) in AVA. In questo caso puoi usare [`nyc`](https://github.com/bcoe/nyc), poichè si tratta sostanzialmente di `istanbul` con il supporto per sub-processi.
+
+Dalla versione `5.0.0` utilizza anche le source maps per la rappresentazione del code coverage per il tuo codice sorgente, a prescindere dai file prodotti dal transpiler. Assicurati che il file che stai testando abbia una source map inline oppure referenzi un file source map. Se utilizzi `babel-register` puoi specificare l'opzione `sourceMaps` come `inline` nella configurazione Babel.
+
+## Domande frequenti
+
+### Perchè non usare `mocha`, `tape`, `node-tap`?
+
+Mocha richiede che tu usi funzioni globali come `describe` o `it` con l'interfaccia di default (usata dalla maggior parte delle persone). Non impone una struttura precisa ed esegue test in serie senza alcun isolamento a livello di processo, rendendolo lento.
+
+Tape e tap sono buoni moduli. AVA ha preso ispirazione dalla loro sintassi. Anche loro eseguono test in serie. Il loro output [TAP](https://testanything.org) integrato non è molto pratico da usare e quindi si finisce sempre per usare un reporter TAP esterno.
+
+Viceversa AVA impone una struttura precisa ed esegue i test in modo concorrente, ogni file di test in un processo separato. Il suo reporter integrato è facile da comprendere a prima vista, nonostante ciò AVA supporta anche l'output TAP mediante il parametro da CLI.
+
+### Come posso utilizzare reporter personalizzati?
+
+AVA supporta il formato TAP ed è quindi compatibile con ogni [TAP reporter](https://github.com/sindresorhus/awesome-tap#reporters). Usa il [parametro `--tap`](#produzione-tap-opzionale) per abilitare l'output TAP.
+
+### Come si scrive e pronuncia AVA?
+
+AVA, no Ava o ava. Si pronuncia [`/ˈeɪvə/` ay-və](https://github.com/sindresorhus/ava/blob/master/media/pronunciation.m4a?raw=true).
+
+### Cos'è lo sfondo del logo in cima?
+
+La [galassia di Andromeda](https://es.wikipedia.org/wiki/Galaxia_de_Andr%C3%B3meda).
```

Comment:
A link in Italian is better : `https://it.wikipedia.org/wiki/Galassia_di_Andromeda`